src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor() {
+    super();
+    this.hidden = false;
+  }
```

Comment:
У тебе травоїдні тварини будуть без імені..
```suggestion
  constructor(name) {
    super(name);

    this.hidden = false;
  }
```